portfolio/src/main/java/com/google/sps/servlets/Logout.java

```diff
@@ -33,16 +33,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.setContentType("text/html");
 
     UserService userService = UserServiceFactory.getUserService();
-    if (userService.isUserLoggedIn()) {
-        String urlToRedirectToAfterUserLogsOut = "/";
-        String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
-        response.getWriter().println("user is now logged out");
-        log.info("user is now logged out");
-        response.sendRedirect(logoutUrl);
-    } else {
-        response.getWriter().println("user is already logged out"); 
-        log.info("user is already logged out");
-        response.sendRedirect("/");       
+    String logMsg = "user is already logged out";
+    String redirectUrl = "/";
+
+    if (!userService.isUserLoggedIn()) {
+    logMsg = "user is now logged out";
+    redirectUrl = userService.createLogoutURL("/");
+    }
+    response.getWriter().println(logMsg);
+    log.info(logMsg);
+    response.sendRedirect(redirectUrl);
```

Comment:
It looks like you are doing almost the exact same logic between Login.java and Logout.java. Could you reduce the duplicate code here (by using a shared helper function maybe?)?

Reply:
The separation makes sense to me because the logMsg is different in every case and .createLogoutURL and .createLoginURL are different methods although the classes do similar things.